connectors/uplynk/build.gradle

```
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.serialization'
+    id 'org.jetbrains.dokka'
+}
```

You need to include the publish script in order to set up the Maven publication.
```suggestion
}
apply from: "$rootDir/connectors/publish.gradle"
```

---

There is a separate ticket for publishing Uplynk connector, I'll apply it there, ok?

---

I didn't spot that. Works for me! 🙂